services/docker.go

```diff
@@ -76,10 +76,9 @@ func ResizeExecConnection(execId string, ctx context.Context, cols, rows uint) e
 
 func CreateInstance(net string, dindImage string) (*Instance, error) {
 
-	var maximumPidLimit int64
-	maximumPidLimit = 150 // Set a ulimit value to prevent misuse
 	h := &container.HostConfig{NetworkMode: container.NetworkMode(net), Privileged: true}
-	h.Resources.PidsLimit = maximumPidLimit
+	h.Resources.PidsLimit = int64(150)
+	h.Resources.Memory = int64(5.12E+8)
```

Comment:
Instead of writing this weird number, how about declaring this:
```go
const (
	BYTE     = 1.0
	KILOBYTE = 1024 * BYTE
	MEGABYTE = 1024 * KILOBYTE
)
```

And then using `512*MEGABYTE` ?